docs/sdks/server-sdks/python.md

```diff
@@ -322,3 +322,130 @@ if self.campaign_json is not None:
 ```
 
 Assuming your service can be configured with many input parameters, that assignment type enables very powerful configuration changes.
+
+## 6. Contextual Bandits
+
+To leverage Eppo's contextual bandits using the Python SDK, there are two additional steps over regular feature flags:
+1. Add a bandit action logger to the assignment logger
+2. Querying the bandit for an action
+
+We have a simple end-to-end example in the [Python SDK repository](https://github.com/Eppo-exp/python-sdk/blob/main/example/03_bandit.py).
+
+### A. Add a bandit action logger to the assignment logger
+
+In order for the bandit to learn an optimized policy, we need to capture and log the bandit actions.
+This requires adding a bandit action logging callback to the AssignmentLogger class
+```
```

Comment:
specify the language after the backticks to get sweet syntax highlighting!
```
```python
```